.github/workflows/main.yml

```
@@ -1,6 +1,12 @@
 name: Lint
 
-on: [push]
+on:
```

I think we can spin this on every pr and push, since it doesn't take long time.

```suggestion
on: [push, pull_request]
```